app/controllers/api/internal/helper/purchases_controller.rb

```diff
@@ -178,15 +178,16 @@ def resend_all_receipts
 
   SEARCH_PURCHASE_OPENAPI = {
     summary: "Search purchase",
-    description: "Search purchase by email, seller, license key, or card details. At least one of the parameters is required.",
+    description: "Search purchase by query (order ID, email, IP, card fingerprint, or external ID), seller email, license key, or card details. At least one parameter is required. Using query with an order ID is the most efficient search method.",
     requestBody: {
       required: true,
       content: {
         'application/json': {
           schema: {
             type: "object",
             properties: {
-              email: { type: "string", description: "Email address of the customer/buyer" },
+              query: { type: "string", description: "Search query - can be order ID (external ID), email, IP address, or card fingerprint" },
+              email: { type: "string", description: "Email address of the customer/buyer (alternative to query)" },
```

Comment:
Oh it's super weird we already have an `email` param but pass it as `query`. Could you remove this?